### PR TITLE
Issue 9142 7

### PR DESCRIPTION
**Pull request by rnveach**
Issue #9142

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java [hunk 1]
```diff
@@ -40,15 +40,15 @@ protected String getPackageLocation() {
     @Test
     public void testGetAcceptableTokens() {
         final RegexpCheck regexpCheck = new RegexpCheck();
-        assertArrayEquals(CommonUtil.EMPTY_INT_ARRAY, regexpCheck.getAcceptableTokens(),
-                "RegexpCheck#getAcceptableTokens should return empty array by default");
+        assertWithMessage("RegexpCheck#getAcceptableTokens should return empty array by default")
+                .that(regexpCheck.getAcceptableTokens()).isEqualTo(CommonUtil.EMPTY_INT_ARRAY);
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java:44]: Is there `isEmpty` for array ?

**Review by romani — APPROVED** [review]
Ok to merge

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java (hunk 1 above)
Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java]: it is compialble:
```
    public void testGetAcceptableTokens() {
        final RegexpCheck regexpCheck = new RegexpCheck();
        assertWithMessage("RegexpCheck#getAcceptableTokens should return empty array by default")
                .that(regexpCheck.getAcceptableTokens())
                .isEmpty();
    }
```

Comment (rnveach) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java]: Fixed. I guess CI won't guide me on this.

**Review by pbludov — APPROVED** [review]

**Pull request merged**